packages/components/src/public/QueryModel/CustomizeGridViewModal.tsx

```diff
@@ -13,12 +13,19 @@ import { QueryModel } from './QueryModel';
 
 // exported for jest testing
 export const includedColumnsForCustomizationFilter = (column: QueryColumn, showAllColumns: boolean): boolean => {
+    const isAncestor = column.fieldKeyPath.indexOf('/Ancestors') >= 0;
```

Comment:
column.fieldKeyPath?.indexOf

Reply:
Sure, good to be defensive here, but are there columns that don't have fieldKeys?

Reply:
Probably not in real scenarios, but would make jest tests happier